tox.ini

```diff
@@ -2,7 +2,7 @@
 skipsdist = True
 usedevelop = True
 
-envlist = py{27,34,35}-dj{18,110,110head,dj11}-{sqlite,postgres,mysql}-{elasticsearch5,elasticsearch2,elasticsearch,noelasticsearch},
+envlist = py{27,34,35}-dj{18,110,110head,111}-{sqlite,postgres,mysql}-{elasticsearch5,elasticsearch2,elasticsearch,noelasticsearch},
```

Comment:
Since Django 1.11 would be the first in supporting python 3.6, should we add it to the ``envlist``?